src/doc/man/cargo-package.md

```diff
@@ -26,9 +26,9 @@ stored in the `target/package` directory. This performs the following steps:
     - The original `Cargo.toml` file is rewritten and normalized.
     - `[patch]`, `[replace]`, and `[workspace]` sections are removed from the
       manifest.
-    - `Cargo.lock` is automatically included if the package contains an
-      executable binary or example target. {{man "cargo-install" 1}} will use the
-      packaged lock file if the `--locked` flag is used.
+    - `Cargo.lock` is included if available, or generated when missing.
```

Comment:
I feel like leading with `if available` will set the tone for readers so that they misunderstand this. I know my mind went one direction when starting to read this and I had to bring it back to where things are with the second half. We need to assume people will stop reading documentation at any point, because they do.

Reply:
The last as it makes a clear statement and then clarifies